en/lessons/specifics/ecto/02_associations.md

```diff
@@ -0,0 +1,672 @@
+---
+version: 1.3.0
+title: Associations
+---
+
+In this section we'll learn how to use Ecto to define and work with associations between our schemas.
+
+## Table of Contents
+
+* Set up
+* Types of associations
+  * Belongs to/has many
+  * Belongs to/has one
+  * Many to Many
+* Saving associated data
+  * `build_assoc`
+  * `put_assoc`
+* Querying for associated data
+  * Preloading data
+
+## Set Up
+
+First, we'll set up a simple app that uses Ecto. These steps are taken from the [Getting Started](https://hexdocs.pm/ecto/getting-started.html#adding-ecto-to-an-application) guide of the official Ecto documentation.
+
+Create a new app with a supervision tree:
+
+```elixir
+mix new moviedb --sup
+```
+
+Add the Ecto and Postgrex dependencies:
+
+```elixir
+# mix.exs
+defp deps do
+  [{:ecto, "~> 2.0"},
+   {:postgrex, "~> 0.11"}]
+end
+```
+
+Install your dependencies with `mix deps.get`.
```

Comment:
Not sure, but I think you should show the how output looks like to users to be sure.

Reply:
@midnio for the sake of brevity we often exclude this output because it _will_ change slightly as PATCH versions are released, which ultimate leads to a lot confusion and/or PRs updating versions.